apps/analogtime/analog_time.star

```diff
@@ -0,0 +1,121 @@
+"""
+Applet: Analog Time
+Summary: Show time analog style
+Description: Shows the time on an analog style clock.
+Author: rs7q5 (RIS)
+"""
+
+#analog_time.star
+#Created 20220204 RIS
+#Last Modified 20220204 RIS
+
+load("render.star", "render")
+load("schema.star", "schema")
+load("math.star", "math")
+load("time.star", "time")
+
+base_URL = "https://nationaltoday.com/what-is-today/"
```

Comment:
Looks good to me other than I don't think this should be in here

```suggestion
```

Reply:
Correct, I guess I forgot to remove it. not sure why that broke lint though.